chat-commands.js

```diff
@@ -128,6 +128,34 @@ exports.commands = {
 			'How many digits of pi do YOU know? Test it out <a href="http://guangcongluo.com/mempi/">here</a>!');
 	},
 
+	code: function (target, room, user) {
+		if (!target) return this.parse('/help code');
+		if (!this.canTalk()) return;
+		const separator = '\n';
+		if (target.includes(separator)) {
+			const params = target.split(separator).map(param => param.replace(/ /g, '&nbsp;'));
```

Comment:
Browsers usually copy nbsp's as real spaces.

I think it might be better to use `white-space: pre-wrap;` than to manually screw around with nbsp's, though.